jina/executors/indexers/keyvalue.py

```diff
@@ -75,6 +78,35 @@ def query(self, key: int) -> Optional[bytes]:
             with mmap.mmap(self.query_handler.body, offset=p, length=l) as m:
                 return m[r:]
 
+    def update(self, keys: Iterator[int], values: Iterator[bytes], *args, **kwargs):
+        # check: hard fail (raises) on key not found
+        missed = []
+        for key in keys:
+            if self.query_handler.header.get(key) is None:
+                missed.append(key)
+        if missed:
+            # FIXME get indexer name
```

Comment:
Anything left to fix here?

Reply:
Wasn't sure if there is a way to get the name of the indexer. What i'm logging here is the path to the file instead